rules/S6901/java/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "\".setDaemon()\", \".setPriority()\" and \".getThreadGroup()\" should not be invoked on virtual threads",
```

I think we should only write the method name without `.`. The `.` does not really belong to the function name but is the member selection operator, for which we do not have a first argument (i.e., the owner of the member).

Is there an example where we did it differently in a rule earlier?

```suggestion
  "title": "Methods \"setDaemon()\", \"setPriority()\" and \"getThreadGroup()\" should not be invoked on virtual threads",
```